test/instrumentation/levee/verify-govet-levee.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script vets all *.go files by running `go vet -vettool=levee`.
+# It is equivalent to `make vet` with appropraite flags.
+# Usage: `test/instrumentation/levee/verify-govet-levee.sh`.
+
+set -o errexit
+set -o pipefail
+
+KUBE_ROOT=$(dirname $(realpath "${BASH_SOURCE[0]}"))/../../../
+BAZEL_OUT_DIR="$KUBE_ROOT/bazel-bin"
```

Comment:
what's the reason for using bazel here?
none of the other verification tools use it. we use it for some large build / unit test for caching purposes, but even that is not necessarily something we're investing in further / may remove.

importantly I don't think we can use bazel here while not polluting vendor / go.mod